agreement.md

```diff
@@ -129,29 +120,52 @@ If a malfunction occurs, but the machine can still be used safely (for example,
 containment.
 10. Spray-painting should be done outside the building without marking any surface.
-11. All waste fluids must be disposed of properly, in proper containers, following EH&S approved disposal practices.
-Only fluids generated by official SCC related activities may be disposed at the SCC.
+11. All waste fluids must be disposed of properly, in proper containers, following Georgia Tech Environmental Health &
+Safety approved disposal practices. Only fluids generated by official SCC related activities may be disposed of at the SCC.
 12. Only water and hand cleansers may be used in the sinks. If it's not water, don't put it down a drain.
-13. The conference room is to be used for discussions, planning, computer modeling, metrology, and vinyl cutting only.
-It is not for fabrication of components of any kind, nor is it a storage area for any team.
-14. The horizontal surface over the entrance hallway is not to be used for storage. The un-fenced areas on the mezzanine
+13. The horizontal surface over the entrance hallway must not be used for storage. The un-fenced areas on the mezzanine
 must remain clear at all times.
-15. Personal or classroom projects must not be a distraction or take up space in any common area of the shop or the
-parking or drive areas outside the shop without explicit permission from Dr. Cunefare. Personal projects must not make
+14. Personal or classroom projects must not be a distraction or take up space in any common area of the shop, or in the
+parking or drive areas outside the shop, without explicit permission from Dr. Mayor. Personal projects must not make
 any significant use of team or SCC consumable resources.
-16. Any time a vehicle is parked in front of the roll-up door, the keys must be inside the building with someone who has
+15. Any time a vehicle is parked in front of the roll-up door, the keys must be inside the building with someone who has
 permission to move the vehicle at a moment's notice. Contact information for that individual must be prominently
 visible through the windshield.
 
 Infractions of these rules may lead to individual and/or team sanctions, up to and including expulsion from the SCC.
 
 If you need clarification on any of these rules, contact [scc@me.gatech.edu][sccrt].
 
+## Student Competition Center Building Access Policy
+
+All teams and visitors within the Student Competition Center ("SCC") must follow these guidelines. Failure to follow
+these guidelines may result in removal of access to the space.
+
+1. Members with BuzzCard access to the building may not let other members into the space, via car gate or door access,
+without getting explicit permission from the security desk, building management, Jacob Tompkins, or Dr. Rhett Mayor.
+2. Every vehicle must buzz in via BuzzCard. Multiple vehicles may not enter the parking lot during the same tap, even
+if the second vehicle has BuzzCard access.
```

Comment:
```suggestion
2. Every vehicle must be granted access to the parking lot by presenting a valid BuzzCard to the card reader. 
Tailgating behind another vehicle to gain entry is prohibited, even if the second vehicle would otherwise have access.
```